git_greppers/utils.sh

```
@@ -0,0 +1,127 @@
+}
+
+# Create log file
+create_log_file() {
+  local output_dir=$1
+  local primary_branch=$2
+  local repo_folder=$3
+  local max_diff_lines=$4
+  
+  local log_file="$output_dir/processing.log"
+  echo "#KEY#BRANCH# $primary_branch" > "$log_file"
+  echo "#KEY#GENERATED_DATE# $(date)" >> "$log_file"
```

Shellcheck (0.10.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)